shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/TxData.hs

```diff
@@ -123,15 +125,30 @@ data StakePoolRelay =
      -- ^ A @SRV@ DNS record
   deriving (Eq, Generic, Show)
 
-instance NoUnexpectedThunks StakePoolRelay
+instance NoUnexpectedThunks StakePoolRelay where
```

Comment:
You probably don't need to write these. The should be derivable.

Reply:
ok, I will orphan the `IPv4` and `IPv6` instances in a new orphan module.